### PR TITLE
style(cake-vault): Remove text decoration from Pools Help button, make 'add to matamask' cursor: pointer

### PR DESCRIPTION
**Pull request by ChefHutch**
Tiny ui changes, PR title says it all

### CODE REVIEW

File: src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ const ExpandedFooter: React.FC<ExpandedFooterProps> = ({
         </Flex>
       )}
       {account && isMetaMaskInScope && tokenAddress && (
-        <Flex justifyContent="flex-end">
+        <Flex justifyContent="flex-end" style={{ cursor: 'pointer' }}>
```
Comment (memoyil) [inline src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx:131]: Can we also add underline when hover ?

Comment (ChefHutch) [reply src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx]: Nice catch, I've added those styles in cheers Memoyil.

**Review by Chef-Cheems — APPROVED** [review]

**Comment by RabbitDoge** [comment]
Isn't it better to replace the `Text` component by `<Button variant='text'>...` ?

**Comment by ChefHutch** [comment]
Nice one @RabbitDoge, I've amended that

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**